numpy/typing/tests/data/reveal/modules.py

```diff
@@ -32,6 +32,8 @@
 reveal_type(np.__path__)  # E: list[builtins.str]
 reveal_type(np.__version__)  # E: str
 reveal_type(np.__git_version__)  # E: str
+reveal_type(np.test)  # E: numpy._pytesttester.PytestTester
+reveal_type(np.test.module_name)  # E: str
```

Comment:
Do these lines add new tests for the new typing?

Reply:
Yes, they're used for validating that mypy correctly infers their type.